src/backend/marsha/core/utils/medialive_profiles/medialive-720p.json

```diff
@@ -33,19 +33,6 @@
 			"LanguageCodeControl": "FOLLOW_INPUT",
```

Comment:
It's hard to understand what you do and why from the commit message:
- "We removed the 2 higher profile in 720p" : do you mean audio profiles?
- "and to close" > "and too close"

Reply:
> "We removed the 2 higher profile in 720p" : do you mean audio profiles?

It's both. We removed the 720p video profiles and their related audio config